packages/blade/src/components/Input/BaseInput/BaseInputAnimatedBorder.native.tsx

```diff
@@ -9,11 +9,15 @@ import Animated, {
 
 import type { ReactNode } from 'react';
 import { useEffect } from 'react';
+import type { BaseInputProps } from './BaseInput';
 import { makeBorderSize } from '~utils';
+import type { Theme } from '~components/BladeProvider';
 import { useTheme } from '~components/BladeProvider';
 import type { ActionStates } from '~tokens/theme/theme';
 
-const BaseInputStyledAnimatedBorder = styled(Animated.View)(({ theme }) => ({
+const BaseInputStyledAnimatedBorder = styled(Animated.View)<{
+  theme: Theme;
```

Comment:
Don't need theme type.

Reply:
fixed in 717f7a1da44ddbaf76edc37e72d60e182d95576b